browser/ui/views/frame/brave_tab_strip_region_view.cc

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/views/frame/brave_tab_strip_region_view.h"
+#include <memory>
+#include <string>
+#include <utility>
+#include "brave/common/pref_names.h"
+#include "chrome/browser/ui/views/frame/browser_view_layout.h"
+#include "chrome/browser/ui/views/tabs/tab_search_button.h"
+#include "chrome/browser/ui/views/tabs/tab_strip_controller.h"
+
+BraveTabStripRegionView::BraveTabStripRegionView(
+    std::unique_ptr<TabStrip> tab_strip)
+    : TabStripRegionView(std::move(tab_strip)) {
+  PrefService* prefs = tab_strip_->controller()->GetProfile()->GetPrefs();
```

Comment:
i would just make this a private variable and access it again in `OnPreferenceChanged`

Reply:
I did try with the private variable and did `tab_strip_ = tab_strip.get()` this didn't work because it was empty. We're transfering ownership with `std::move`